battery

```diff
@@ -75,34 +67,51 @@ battery_charge() {
             fi
             ;;
         "Linux")
-            case $(cat /etc/*-release) in
+            resolve_bat_state_file_prefix() {
+                local prefixes prefix
```

Comment:
Could you elaborate on what are you doing here? 😃

Reply:
This is the whole reason for issue #39 - on some hardware the `_full` & `_now` files have different prefixes. This simple function finds out which prefix is used; haven't encountered any other prefixes than those two, so it checks against `prefixes=(charge energy)`.

If files `charge_full` & `charge_now` are found under `$battery_path`, then it returns `charge` as prefix; `energy` otherwise. If neither could be resolved, then function returns error code and script exits from line 84.